app/mailers/send_email.rb

```diff
@@ -0,0 +1,6 @@
+class SendEmail < ApplicationMailer
+  def send_help_email(help)
+    @help = help
+    mail to: "mykhaylo.yusko@gmail.com", subject: "Hello"
```

Comment:
`to` should be dynamic. This email should come with help_request form. `subject` as well